invenio_oauthclient/contrib/globus.py

```diff
@@ -79,8 +80,6 @@
 class GlobusOAuthSettingsHelper(OAuthSettingsHelper):
     """Default configuration for Globus OAuth provider."""
 
-    external_method = "globus"
```

Comment:
I have removed all these hardcoded values, and replaced with `remote.name`, otherwise you cannot have 2 Globus apps configured (or others)... which might happen in weird cases! :)